### PR TITLE
45 resturcture docs

### LINKED ISSUE #45: Resturcture docs
A lot of information about running the pipelines is present in the README. Move this information into `docs/source/guides` and update the README to only include general information about the project.

### PR DESCRIPTION
**Pull request by 1ndy**

### CODE REVIEW

File: docs/getting_started.rst [hunk 1]
```diff
@@ -0,0 +1,52 @@
+Getting started
+===============
+
+The Enzyme Function Initiative (EFI) offers several different tools to help you
+identify isofunctional protein families. This guide will walk you through the
+initial setup of the tools
```
Comment (nilsoberg) [inline docs/getting_started.rst]: End with period?

File: docs/getting_started.rst [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+~~~~~~~~~~~~~~~~~~~
+
+If running without docker, dependencies will need to be installed manually
+along with the python packages in ``requirements.txt`` and Perl modules in
+``cpanfile``. 
+
+For software dependencies, see the installation guides for each tool. Then for
+library dependencies:
+
+1. We recommend creating a python virtual environment before installing dependencies::
```
Comment (nilsoberg) [inline docs/getting_started.rst:38]: Is a space needed before ::?

File: docs/getting_started.rst [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+        python -mvenv efi-env
+
+2. Once that command completes, activate the environment::
```
Comment (nilsoberg) [inline docs/getting_started.rst:42]: Is a space needed before ::?

File: docs/getting_started.rst [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+        source efi-env/bin/activate
+
+   and install the dependencies ::
+
+        pip install -r requirements.txt
+
+3. Then install the Perl dependencies::
```
Comment (nilsoberg) [inline docs/getting_started.rst:50]: Is a space needed before ::?

File: docs/source/pipelines/est/index.rst [hunk 5]
```diff
@@ -90,4 +90,87 @@ submission script called ``run_nextflow.sh``: ::
...
+
+Execution Details
+-----------------
+
+.. .. image:: images/pipelines/est.png
+..    :alt: Visualization of EST pipeline
+
+The EST pipeline consists of different stages which transform the input
+sequences into network edges. The stages are executed roughly in this
+chronological order
```
Comment (nilsoberg) [inline docs/source/pipelines/est/index.rst]: End with :?

**Review by nilsoberg — CHANGES_REQUESTED** [review]
There may be some minor documentation syntax issues to address.

File: docs/getting_started.rst (hunk 2 above)
Comment (1ndy) [reply docs/getting_started.rst:38]: No, this is equivalent to
```
installing dependencies: ::
```
https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#literal-blocks

**Review by nilsoberg — APPROVED** [review]

**Pull request merged**